### PR TITLE
BEAM-3368 fix translation for external

### PR DESCRIPTION
**Pull request by wcn3**
Add a translation for beam.External trasnforms.

Follow this checklist to help us incorporate your contribution quickly and easily:

---

### CODE REVIEW

File: sdks/go/pkg/beam/core/runtime/graphx/translate.go [hunk 1]
```diff
@@ -238,6 +239,9 @@ func (m *marshaller) makePayload(edge *graph.MultiEdge) *pb.FunctionSpec {
 	case graph.GBK:
 		return &pb.FunctionSpec{Urn: urnGBK}
 
+	case graph.External:
+		return &pb.FunctionSpec{Urn: urnExternal}
```
Comment (herohde) [inline sdks/go/pkg/beam/core/runtime/graphx/translate.go]: Should be: return &pb.FunctionSpec{Urn: edge.Payload.URN, Payload: edge.Payload.Data}

File: sdks/go/pkg/beam/core/runtime/graphx/translate.go [hunk 2]
```diff
@@ -33,12 +33,13 @@ import (
...
-	urnGBK     = "urn:beam:transform:groupbykey:v1"
-	urnCombine = "urn:beam:transform:combine:v1"
-	urnWindow  = "urn:beam:transform:window:v1"
+	urnImpulse  = "urn:beam:transform:impulse:v1"
+	urnParDo    = "urn:beam:transform:pardo:v1"
+	urnFlatten  = "urn:beam:transform:flatten:v1"
+	urnGBK      = "urn:beam:transform:groupbykey:v1"
+	urnCombine  = "urn:beam:transform:combine:v1"
+	urnWindow   = "urn:beam:transform:window:v1"
+	urnExternal = "urn:beam:transform:external:v1"
```
Comment (herohde) [inline sdks/go/pkg/beam/core/runtime/graphx/translate.go]: (and this won't be needed)

File: sdks/go/pkg/beam/core/runtime/graphx/translate.go (hunk 1 above)
Comment (wcn3) [reply sdks/go/pkg/beam/core/runtime/graphx/translate.go]: Done.

File: sdks/go/pkg/beam/core/runtime/graphx/translate.go (hunk 2 above)
Comment (wcn3) [reply sdks/go/pkg/beam/core/runtime/graphx/translate.go]: Done.

**Comment by wcn3** [comment]
@lukecwik please merge.

**Pull request merged**